bot/models/migrations/versions/e2101b941963_alter_user_collumn_to_currency.py

```diff
@@ -23,3 +23,4 @@ def upgrade():
 
 def downgrade():
     op.alter_column('user', 'currency', nullable=False, new_column_name='xp_points')
+    op.drop_column('user', sa.Column('daily_last_collected_at', sa.DateTime, nullable=True))
```

Comment:
Opa, pela doc, acho que seria:
```suggestion
    op.drop_column('user', 'daily_last_collected_at')
```